src/frontend/src/index.ts

```diff
@@ -48,11 +49,23 @@ const printVersion = () => {
   if (version.dirty) {
     console.warn("This version is dirty");
   }
+
+  if (anyFeatures()) {
+    const featuresRec: Record<string, boolean> = features;
```

Comment:
not sure wht's the purpose of this `featuresRec`?

Reply:
none 🙃